two_factor/utils.py

```diff
@@ -6,12 +6,12 @@
 USER_DEFAULT_DEVICE_ATTR_NAME = "_default_device"
 
 
-def default_device(user):
+def default_device(user, confirmed=True):
```

Comment:
The changes here are partially covered by the test added in `tests/test_email.py` but I think we could do with some more thorough testing, e.g. test that `default_device(user)` returns only confirmed email devices and `default_device(user, None)` returns both confirmed and unconfirmed.

Reply:
Just added 2 new tests to address this. Let me know what you think.